Improvements to class file and sample content

compwiztobe opened the pull request:
Several things here, including:
- correct page numbering and styling from abstract to references, empty pagestyle afterwards (there were many bugs)
- tweaking spacing on title pages for appearance, fonts, etc.
- additional features for CV
- grammar, date formats
- nicer chapter headings and page headers
- more sample content to verify formats (Korean and English, different types of references, with webpage support)
- etc.

Not to mention repairing file format to Unix standard (line ending, spaces for indent, ...).  Since this confounds the PR diff, see this diff instead for all the other changes to the class file:
```
git diff 76523f8 improve-format
```

compwiztobe committed:
fix gitignore

compwiztobe committed:
fix indentation, line endings, excessive whitespace, etc.

compwiztobe committed:
fixing up the class file

fixing lots of whitespace and punctuation issues around macro expansions
tweaking spacing and other formatting
removing ugly sans serif fonts
correcting grammar
ISO date format (also correcting British locale date suffixes)
fixing page number omission in acknowledgements and CV (in the multipage case, and last page of references)
fitting CV on one page
bringing english summary into sync with korean summary

compwiztobe committed:
make korean title and author optional

compwiztobe committed:
several more fixes

twopage format with nicer chapter titles beginning only on odd pages
some better title page text wrap
remove more strange fonts
correct front matter page numbering
fix summary environments

compwiztobe committed:
midchapter pages put page number and chapter title in the header

also need to fix the chapter* environments to properly clear the page after the content,
so that later chapter* commands don't ruin the pagestyle on the previous page

compwiztobe committed:
update sample content and citations to fill out special sections and check format

Korean lipsum package jiwonlipsum from
https://github.com/ChengBinJin/LaTex-beginner-lecture/blob/master/jiwonlipsum.sty
no longer available at CTAN?

compwiztobe committed:
initial commit of unsrturl-inline.bst, generated with urlbst

modified version of unsrturl bibtex style, to include inline links, generated with
$ /usr/share/texmf-dist/scripts/urlbst/urlbst --inlinelinks --hyperref /usr/share/texmf-dist/bibtex/bst/base/unsrt.bst unsrturl-inline.bst

this proved simpler than generating a whole new bst file from scratch with the custom-bib package
https://ctan.org/pkg/custom-bib

compwiztobe committed:
tweak unsrturl-inline to explicitly show urls for webpage entries, with modified formatting

compwiztobe committed:
better spacing on degree approval page to better match all departments

Commit:
Jaehun Shin's publication and subpublication listing environments

lonelywing commented:
Sorry for the late check.

Firstly, I sincerely appreciate your contributions on cleaning the messy code to nicer project.

I can accept your pull-request only when the format of the main contents satisfy the format recommended by the univ.

Please see updated file named: 대학원-학위논문-작성-지침_2020.10.29.pdf


For example, the title of the page 1, introduction must be center aligned, whereas the pull-requested version show left aligned title.

The page number must also be positioned on the bottom center of the page with hyphens. However, pull-requested version show page number on the left-top side without hyphens. The page style from first page and the second page is also inconsistent.

I kinda skimmed through your contribution, and these are some points that needs change to keep the regulation in the format suggested from the university. 

I'll list up all the points that needs modification within few weeks, based on the 대학원-학위논문-작성-지침_2020.10.29.pdf.


Many thanks in advance.

compwiztobe committed:
class file imports scanned approval pages if provided

and correctly clears double pages in the approval section for twosided documents

compwiztobe committed:
revert chapter heading (first page) changes

this partially reverts commit 51afb11fa0dd95bb385323e4381dd8ca22275632.

compwiztobe committed:
revert chapter header changes (on mid-chapter pages)

this partially reverts commit 1fc7e0369b7a1bf2477959274a6bbded63bc189a.

compwiztobe committed:
add dedication page
